check-submodule.py

```diff
@@ -229,6 +238,54 @@ def save_diff_files(
     asyncio.run(json_diff.diff())
 
 
+# Compute continuous integration metrics before and after a
+# tree-sitter-language update.
+def compute_ci_metrics(args: argparse.Namespace) -> None:
+
+    if args.language not in EXTENSIONS.keys():
+        print(args.language, "is not a valid tree-sitter-language")
+        sys.exit(1)
+
+    # Repository passed as input
+    repo_dir = pathlib.Path(args.path)
+
+    # Create rust-code-analysis repository path
+    rca_path = WORKDIR / "rust-code-analysis"
+
+    # Old metrics directory
+    old_dir = WORKDIR / (args.language + OLD_SUFFIX)
+    # New metrics directory
+    new_dir = WORKDIR / (args.language + NEW_SUFFIX)
+
+    # Create output directories
+    old_dir.mkdir(parents=True, exist_ok=True)
+    new_dir.mkdir(parents=True, exist_ok=True)
+
+    # Git clone rust-code-analysis master branch repository
+    print(f"Cloning rust-code-analysis master branch into /tmp")
+    run_subprocess(
+        "git",
+        "clone",
+        "--depth=1",
+        "--recurse-submodules",
+        "-j8",
+        "https://github.com/mozilla/rust-code-analysis",
+        rca_path,
+    )
```

Comment:
We already have a local clone, we can just reuse it by checking out to master instead of recloning.

Reply:
Some CI systems fetch a single branch and you need to reset commits and make complicated stuff to retrieve master branch's commits. This solution could potentially be used on any CI system.